src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ReportHelper.java

```diff
@@ -93,7 +96,7 @@ public static ListingData processClaimsAcceptedRequest(ListingDetails listingDet
             List<AdhocReportTypeItem> localReportsDetailList = new ArrayList<>();
             for (SubmitEvent submitEvent : submitEvents) {
                 AdhocReportTypeItem localReportsDetailItem =
-                        getClaimsAcceptedDetailItem(listingDetails, submitEvent.getCaseData());
+                        getClaimsAcceptedDetailItem(listingDetails, submitEvent);
```

Comment:
Would it be better to check the state before this logic? That way we aren't creating and returning the empty object in line 175 and saves a bit on memory